src/status_im2/contexts/chat/lightbox/zoomable_image/view.cljs

```diff
@@ -0,0 +1,313 @@
+(ns status-im2.contexts.chat.lightbox.zoomable-image.view
+  (:require
+   [react-native.core :as rn]
+   [react-native.gesture :as gesture]
+   [react-native.reanimated :as reanimated]
+   [reagent.core :as reagent]
+   [utils.re-frame :as rf]))
+
+;; Definitions
+(def min-scale 1)
+
+(def double-tap-scale 2)
+
+(def max-scale 5)
+
+(def init-offset 0)
+
+(def velocity-factor 0.25)
+
+(def default-duration 300)
+
+;; Some aliases for reanimated methods, as they are used 10s of times in this file
```

Comment:
Comments like these are non-idiomatic in Clojure and even Lisps dare I say. You either create sections with triple/quadruple semicolons and add normal comments below and/or you document the functions with docstrings. But double-semicolon comments at the top-level should be avoided from this stylistic perspective. It's in the Clojure Style Guide too https://guide.clojure.style/#comments.

These sections are also valuable to Emacs users of the outline minor mode, because they can be narrowed, collapsed, etc.

TL;DR: Use 3 or 4 semicolons to separate different sections of the namespace.

Reply:
Thanks @ilmotta